packages/lib/src/messaging/Messaging.ts

```diff
@@ -154,19 +202,26 @@ export async function getMessages(
         (
             await getNewMessages(connection, userDb, contact)
         )
-            .filter((envelop) =>
-                envelop.deliveryServiceIncommingTimestamp ? true : false,
-            )
+            .filter((envelop) => {
```

Comment:
Why do we need to filter messages without an incommingTimestamp?

Reply:
I think it was because the specification requires a `deliveryServiceIncommingTimestamp,` but I'm not sure if that was the main reason.

Reply:
As discussed on a call: Remove filter